src/platforms/go/common/troubleshooting.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Troubleshooting
+description: "Troubleshooting common Go Issues"
+keywords: ["go", "troubleshooting"]
+sidebar_order: 1000
+supported:
+  - go
+  - go.echo
+  - go.fasthttp
+  - go.gin
+  - go.http
+  - go.iris
+  - go.martini
+  - go.negroni
+---
+
+## I'm using Go and I don't see a stack trace
+
+Since the Go SDK tries to read the source files from your local disk, it could be possible that you’re compiling your binary and deploying it in a manner that makes the source code inaccessible. 
```

Comment:
```suggestion
Since the Go SDK tries to read the source files from your local disk, it's possible that you’re compiling your binary and deploying it in a manner that makes the source code inaccessible. 
```